### DISCUSSION TITLE
这是不是被伪装的客户端吸血了？

### DISCUSSION THREAD

**Discussion by CAESIUS-TIM** [c0re100/qBittorrent-Enhanced-Edition#566] [Q&A] [answered]
![QQ_1721218724165](https://github.com/user-attachments/assets/6935c3be-faeb-49d4-89dd-870bd8150165)
![image](https://github.com/user-attachments/assets/fc119b57-a8b3-4456-b625-0e32a949f31a)
500MB的文件上传给它了8GB

**Comment by CAESIUS-TIM** [comment, marked as answer]
https://github.com/c0re100/qBittorrent-Enhanced-Edition/discussions/548#discussioncomment-9433540

**Comment by SeaHOH** [comment]
这货前两天刚开始的
